resources/views/pages/orders/add.blade.php

```diff
@@ -0,0 +1,65 @@
+@extends('layouts.master')
+@section('title','Admin | Orders | Add')
+@section('content')
+<!-- Content Wrapper. Contains page content -->
+<div class="content-wrapper">
+    <!-- Content Header (Page header) -->
+    <div class="content-header">
+        <div class="container">
+            <div class="row mb-2">
+                <div class="col-sm-6">
+                    <h1 class="m-0">{{ __('Add Order') }}</h1>
+                    @if ($message = Session::get('success'))
+                    <div class="alert alert-success">
+                        <p>{{ __($message) }}</p>
+                    </div>
+                    @endif
+                </div><!-- /.col -->
+            </div><!-- /.row -->
+        </div><!-- /.container-fluid -->
+    </div>
+    <!-- /.content-header -->
+
+    <!-- Main content -->
+    <div class="content">
+        <div class="container">
+            @if($errors->any())
+                @foreach($errors->all() as $error)
+                    <div class="alert alert-danger">{{$error}}</div>
```

Comment:
```suggestion
                    <div class="alert alert-danger">{{ $error }}</div>
```
fix hết chỗ tương tự nhé